curriculum/challenges/english/blocks/learn-essential-courtesies-at-the-office/68902bdd7751fb8236d4aa57.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 68902bdd7751fb8236d4aa57
+title: Task 3
+challengeType: 22
+dashedName: task-3
+---
+
+<!-- (Audio) A: 你好 -->
+
+# --description--
+
+When you combine the two characters `你(nǐ)` and `好(hǎo)`, it becomes `你好(nǐ hǎo)`, which means hello and is used to greet someone politely.
+
+This is the first word you are learning in Chinese. It can be used in almost any situation, such as at work, at school, or when meeting someone for the first time.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+  `nǐ  hǎo`
+
+`BLANK BLANK`
```

Comment:
I'm wondering if we should review the fill-in-the-blank challenges again and update them to follow the convention. Or do we want to merge this PR first (to complete the chapter structure conversion) and revisit them later?

(Sorry, I know these challenges were written before our convention discussion 😅)